rnd/autogpt_builder/src/components/providers/actions.ts

```diff
@@ -0,0 +1,10 @@
+"use server";
+
+import getServerUser from "@/hooks/getServerUser";
+import AutoGPTServerAPIServerSide from "@/lib/autogpt-server-api/clientServer";
+
+export default async function logPageViewAction(page: string, data: any) {
+  console.debug("logPageViewAction", page, data);
+  const apiClient = new AutoGPTServerAPIServerSide();
+  await apiClient.logPageView({ page, data });
```

Comment:
Will page-view volume eventually be too much for our db on high page access ?
Since GA already covers this along with ip/geographical location, can we utilize it instead? 
We can combine GA data and our DB in looker studio for reporting.

Reply:
I think we can consider that problem when we hit it. Postgres can hold billions of records without problem. When we hit that we are doing good and will probably want different analytics